src/core/services/theming/theming.js

```diff
@@ -670,9 +670,13 @@ function ThemingProvider($mdColorPalette, $$mdMetaProvider) {
    * @param {object} options Theme definition options
    * Options are:<br/>
    * - `primary` - `{string}`: The name of the primary palette to use in the theme.<br/>
+   * - `primaryHues` - `{object} (Optional)`: Overwrite hues for primary palette.<br/>
```

Comment:
Please use `=` at the end of the type to indicate that it is optional, i.e. `{object=}`.

Let's use "Override" instead of "Overwrite".